### PR TITLE
packtpub: add page

### PR DESCRIPTION
**Pull request by bl-ue**
cc @vladimyr

### CODE REVIEW

**Comment by vladimyr** [comment]
Um, ok? :shrug:

File: pages/common/packtpub.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Download the daily offer book into the specified directory:
+
+`packtpub download --dir {{path/to/directory}}`
+
+- Start interactive login to packtpub.com:
+
+`packtpub login`
+
+- Logout of packtpub.com:
```
Comment (sbrl) [inline pages/common/packtpub.md:18]: Perhaps this will read better?
```suggestion
- Logout from packtpub.com:
```

File: pages/common/packtpub.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Download the daily offer book into the current directory with the specified book format (defaults to `pdf`):
+
+`packtpub download --type {{pdf|ebup|mobi}}`
+
+- Download the daily offer book into the specified directory:
+
+`packtpub download --dir {{path/to/directory}}`
+
+- Start interactive login to packtpub.com:
```
Comment (sbrl) [inline pages/common/packtpub.md:14]: Perhaps this will read better?
```suggestion
- Start an interactive login  to packtpub.com:
```

**Review by navarroaxel — APPROVED** [review]

File: pages/common/packtpub.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Download the daily offer book into the current directory with the specified book format (defaults to `pdf`):
+
+`packtpub download --type {{pdf|ebup|mobi}}`
+
+- Download the daily offer book into the specified directory:
+
+`packtpub download --dir {{path/to/directory}}`
+
+- Start an interactive login to packtpub.com:
```
Comment (nicokosi) [inline pages/common/packtpub.md:14]: What about? (not sure if "good english"). It seems obvious you log in to `packtpub.com`. 😇
```suggestion
- Log in interactively:
```

File: pages/common/packtpub.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Logout from packtpub.com:
+
+`packtpub logout`
+
+- Display the daily offer:
+
+`packtpub view-offer`
+
+- Open the the daily offer in the default web browser:
```
Comment (nicokosi) [inline pages/common/packtpub.md:26]:
```suggestion
- Open the daily offer in the default web browser:
```

File: pages/common/packtpub.md (hunk 3 above)
Comment (bl-ue) [reply pages/common/packtpub.md:14]: Yeah, it is now, but what about a user who has the command installed (perhaps he automated the installation of many) and it now seeing what it is.  If he know about packtpub.com he'll know what the command's for, but if he's not heard of the site before he might wonder what it logs in to.

Do you see my logic behind this wording?

**Comment by Waples** [comment]
I haven't tried this program yet, but I also had a bot running in the past that would send me the free books from packtpub, but they have changed the inner workings of free books (downloading) on their servers a year or so ago, which also broke my bot / program. So, is this still relevant @bl-ue ?

**Comment by bl-ue** [comment]
Hmmm, that's not good 😄 

Well, I can't really say.  I doubt @vladimyr is interested in it anymore, so he's probably not going to fix it, but probably plenty of our pages document commands that don't work anymore 🤷🏻 

Anyway, great catch @Waples, thank you for taking the time to review this!  I need @waldyrious's / @sbrl's experienced input here, not sure what to do myself.

**Comment by vladimyr** [comment]
What makes you think I'm not interested in fixing a bug if it exists? 🙃

I'm just not using it personally atm but I'm very much willing address any bug reports.

**Comment by bl-ue** [comment]
Well that's a turn! Honestly, seeing that the [latest commit](https://github.com/vladimyr/packtpub-cli/commit/9dda079b433c8fd429dfb0c35430c7eee0b52cca) was made in 2018 I thought you had lost interest in it.  However, I failed to realize that it was pinned on your profile!

@Waples or @vladimyr will you please test it?  I'd rather not create an account on packtpub.com right now...

**Comment by Waples** [comment]
tbh @bl-ue, I'm not really here to test / debug vladimyr's program :P

File: pages/common/packtpub.md (hunk 4 above)
Comment (sbrl) [reply pages/common/packtpub.md]: Applied change because this has been sitting around a while and it's an obvious typo.

**Review by sbrl — APPROVED** [review]
Thanks, @bl-ue!

File: pages/common/packtpub.md (hunk 4 above)
Comment (bl-ue) [reply pages/common/packtpub.md]: Yes, I for some reason forgot to apply it, though I intended to twice!?! 🤷🏻 😄

**Pull request merged**